pkg/console/subresource/configmap/configmap.go

```diff
@@ -18,23 +19,57 @@ const (
 	consoleConfigYamlFile   = "console-config.yaml"
 	clientSecretFilePath    = "/var/oauth-config/clientSecret"
 	oauthEndpointCAFilePath = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
-	// TODO: should this be configurable?  likely so.
-	documentationBaseURL = "https://docs.okd.io/4.0/"
-	brandingDefault      = "okd"
 	// serving info
 	certFilePath = "/var/serving-cert/tls.crt"
 	keyFilePath  = "/var/serving-cert/tls.key"
 )
 
-func DefaultConfigMap(cr *operatorv1.Console, rt *routev1.Route) *corev1.ConfigMap {
+// overridden by console config
+const (
+	defaultLogoutURL = ""
+)
+
+// overridden by operator config
+const (
+	defaultDocumentationBaseURL = "https://docs.okd.io/4.0/"
+	defaultBranding             = "okd"
+)
+
+func getLogoutRedirect(consoleConfig *configv1.Console) string {
+	if len(consoleConfig.Spec.Authentication.LogoutRedirect) > 0 {
```

Comment:
I believe this is all we need to do.  The `config` structs are not pointers, so when the YAML is instantiated, nested structs should be created with empty string values.  Therefore if empty string is provided, we can return the default value instead.

@jhadvig @zherman0 work?  Same treatment for all three of these.